core/themes/seven/style.css

```diff
@@ -204,45 +215,47 @@ ul.primary li a:visited,
 ul.primary li a:hover,
 ul.primary li.active a {
   display: block;
-  float: left; /* LTR */
-  height: 2em;
+  height: 2.6em;
+  padding: 1px 20px 2px;
+  border: none;
   line-height: 2.6;
-  padding: 0 18px 8px;
-  background-color: #a6a7a2;
-  color: #000;
-  font-weight: bold;
-  border-width: 1px 1px 0 1px;
-  border-style: solid;
-  border-color: #a6a7a2;
-  -moz-border-radius: 8px 8px 0 0;
-  border-radius: 8px 8px 0 0;
-}
-[dir="rtl"] ul.primary li,
-[dir="rtl"] ul.primary li a:link,
-[dir="rtl"] ul.primary li a.active {
-  float: right;
+  color: #757575;
+  font-weight: normal;
+  background-color: #DBDBD9;
+  transition:
+    background-color 0.5s ease 0s,
+    color 0.5s ease 0s;
```

Comment:
This long declaration matches our coding standards (I had to check): https://api.backdropcms.org/css-standards

Nice job!